### PR TITLE
feat: Add workspaces documentation

### LINKED ISSUE #254: Feature: Add workspaces to docs
### Type of feature

🍕 Feature

### Current behavior

We're adding a new feature called workspaces 🚀 

### Suggested solution

- add a feature page
- update guides with info about how to use workspaces.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by BekahHW**
## Description

- Changes the terminology insights -> Repository Insights. Lists -> Contributor Insights
- Adds feature section for workspaces
- Updates sidebar with clearer organization and wording

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

closes #254 #253 
## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by BekahHW** [comment]
## Notes for Updates
- public by default 
- root of the website, they'll be redirected to personal workspace (if you haven't been onboarded)
- close #253 with this issue
- everytime you go back to a workspace, it's the last one that you've accessed

## Paid v. free features
- Private workspaces are paid features


### Contributors
- tracked contributors

File: docs/features/contributor-insights.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+keywords:
+  - "contributor insights"
+  - "contributors"
+  - "organize"
+  - "Monitor Contributors"
+  - "Organize Contributors"
+  - "Contributor Contributor Insights  Management"
+---
+
+The Contributor Insights  feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Contributor Insights  feature, gain granular insights into each contributor's activity and contributions.
```
Comment (jpmcb) [inline docs/features/contributor-insights.md:14]: nit: looks like some extra whitespace here

```suggestion
The Contributor Insights feature enables you to categorize, monitor, and analyze different groups of contributors within open source projects. With the Contributor Insights feature, gain granular insights into each contributor's activity and contributions.
```

File: docs/features/contributor-insights.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Monitor individual commit histories, track who is actively engaged, identify alumni or new contributors, and even compare the performance of contributors against each other. This encompasses a wide range of metrics such as code commits, created and reviewed Pull Requests (PRs), as well as issue creation and commenting.
+
+![Contributor Insights -page](../../static/img/Contributor Insights -page.svg)
+
+## How to Create a Contributor Insight Page
+
+After logging in, navigate to your [Contributor Insights  Page](https://app.opensauced.pizza/hub/Contributor Insights /).
```
Comment (jpmcb) [inline docs/features/contributor-insights.md]: nit: whitespace

```suggestion
After logging in, navigate to your [Contributor Insights Page](https://app.opensauced.pizza/hub/Contributor Insights /).
```

that link also looks broken to me - going there on beta doesn't work. Maybe we need a redirect to the users personal workspace and to create new insight pages.

File: docs/features/contributor-insights.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+There are three ways to add a list:
+
+1. **Explore Contributors**: Use our explore tool to find contributors and create your list.
+2. **Sync your GitHub Team**: Connect to your GitHub to create a Contributor Insight Pagefrom a team in your organization.
```
Comment (jpmcb) [inline docs/features/contributor-insights.md:27]:
```suggestion
2. **Sync your GitHub Team**: Connect to your GitHub to create a Contributor Insight Page from a team in your organization.
```

File: docs/features/contributor-insights.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+3. **Import your GitHub Following**:
+   Connect to your GitHub to create a Contributor Insight Pagewith all the contributors you follow.
```
Comment (jpmcb) [inline docs/features/contributor-insights.md:29]:
```suggestion
   Connect to your GitHub to create a Contributor Insight Page with all the contributors you follow.
```

File: docs/features/contributor-insights.md (hunk 2 above)
Comment (jpmcb) [reply docs/features/contributor-insights.md]: Already fixed!

**Comment by BekahHW** [comment]
Thanks, John! Updated.

**Review by jpmcb — APPROVED** [review]
Looks great!

**Pull request merged**